basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -69,6 +69,9 @@ namespace basic_autonomy
                                                                 const carma_wm::WorldModelConstPtr &wm, cav_msgs::VehicleState &ending_state_before_buffer,
                                                                 const GeneralTrajConfig &general_config, const DetailedTrajConfig &detailed_config, std::unordered_set<lanelet::Id> &visited_lanelets)
         {
+            //TODO: ending_state_before_buffer is unused; if there are no plans for it, then remove from arg list
```

Comment:
yes this can be removed from the arg list without any issues. 
ending_state_before_buffer is used to store the state of the ending downtrack of the maneuver before a buffer is added to trajectory. 
This is being done in the add_lanefollow_buffer function

Reply:
Done.